sspl_test/alerts/node/test_node_bmc_interface.py

```diff
@@ -43,21 +43,25 @@
         simulate_bmc_interface_alert.kcs_channel_alert(BMC_IF_CONSUL_KEY,BMC_IF_CONSUL_VAL)
 
     bmc_interface_message = None
-    time.sleep(25)
-    while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-        time.sleep(0.1)
```

Comment:
We still need this line inside while loop.

Reply:
I didn't remove any lines except time.sleep.